eslint.config.js

```diff
@@ -6,7 +6,7 @@ import globals from "globals";
 import tseslint from "typescript-eslint";
 
 export default tseslint.config(
-  { ignores: ["dist", "src-tauri"] },
+  { ignores: ["dist", "src-tauri", "src/bindings"] },
```

Comment:
Nice, yeah if we import types from `@/bindings` that would work well I think! It looks like they're all in global scope so we just need to make sure not to duplicate names, which seems fine.

Looks like we can manually do `#[ts(rename = "..")]` in ts-rs as a workaround for name conflicts